resources/public/api/conf/2.0/application.yaml

```diff
@@ -808,6 +815,13 @@ paths:
           schema:
             type: string
             example: GB1234567890
+        - in: query
+          name: movementReferenceNumber
+          description: This is an optional query parameter that allows a list of departures and or arrivals to be filtered by an MRN.

```

Comment:
As this is on the arrivals endpoint, this one should only mention arrivals.

Reply:
Done